src/commands/postnsave.js

```diff
@@ -0,0 +1,154 @@
+// @flow
+
+import FeedParser from 'feedparser';
+import request from 'request';
+import fs from 'fs';
+
+import {owner} from '../config/config.json';
+
+import type {Message} from 'discord.js';
+import type {Stash} from '../type/Stash';
+
+let url = 'https://www.craneanime.com/wp/feed/atom';
+// let url = 'https://xkcd.com/atom.xml';
+
+export default {
+  name: 'postnsave',
+  description: 'Check the atom feed',
```

Comment:
Should be `Check the feed and make announcement for new post`?

Reply:
oh i think i skipped that, will change it